OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,20 +1,190 @@
+
+
+Data volumes are designed to persist data, independent of the container’s lifecycle. he default location of the volume in container is under "/var/lib/docker/volumes".
+There is an option to mount a directory from your Docker engine’s host into a container as volume. In this project we will use that option for the data volume. 
+The volume will be used to store Database datafiles and Weblogic server domain files.
+This volume will be created in "/scratch/DockerVolume/SOAVolume/" 
+
+Since the volume is created as "root" user, provide read/write/execute permissions to "oracle" user (by providing permissions to "others"), as all operations inside the container happens with "oracle" user login.
+
+   $ chmod -R 777 /scratch/DockerVolume/SOAVolume
+
+## Database
+
+
+You need to have a running database container or a database running on any machine. 
+The database connection details are required for creating SOA specific RCU schemas while configuring SOA domain. 
+While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
+
+Create an environment file db.env.list
+
+    ORACLE_SID=<DB SID>
+    ORACLE_PDB=<PDB ID>
+    ORACLE_PWD=<password>
+    
+Sample Data will look like this...
+
+    ORACLE_SID=soadb
+    ORACLE_PDB=soapdb
+    ORACLE_PWD=Welcome1
+    
+The Database 12.2.0.1 container can be started from the GitHub Location. 
+Follow the instructions to create a 12.2.0.1 based enterprise edition database.
+
+Sample Command to Start the Database is as follows
+
+     $ docker run --name soadb  --network=SOANet -p 1521:1521 -p 5500:5500 -v /scratch/DockerVolume/SOAVolume/DB:/opt/oracle/oradata --env-file ./db.env.list  oracle/database:12.2.0.1-ee
+The above command starts a DB container attaching to a network and mounting a host directory as /opt/oracle/oradata for persistence. 
+It maps the containers 1521 adn 5500 port to respective host port such that the services can be accessible outside of localhost.
+
+
+## SOA Distributable version 12.2.1.3 Docker image Creation and Running
+
+To build a SOA image either you can start from building Oracle JDK and Oracle FMW Infrastrucure image or use the already available Oracle FMW Infrastructure image.
+The FMW Infrastructure image is available in the container registry and can be pulled from there. If plan to use available Infrastructure image, skip next two steps and jump to "Building Docker Image for SOA"
+
+
+## Building Oracle JDK (Server JRE) base image
+
+Download the Oracle Server JRE binary and drop in folder OracleJava/java-8 and build that image.
+
+    $ cd OracleJava/java-8
+    $ sh build.sh
+NOTE: The files to build JDK image can be found under docker/OracleJava
+
+
+## Building Oracle FMW Infrastructure Docker Install Image
+
+Download the binary of FMW infrastructure and build the image using "buildDockerImage.sh" script for WebLogic.You need to choose the version as "12.2.1.3" and distribution as "infrastructure".
```

Comment:
I would just link directly to the GitHub instructions for building the FMW Infrastructure image.